packages/office-ui-fabric-react/src/components/Link/Link.styles.ts

```diff
@@ -14,12 +15,12 @@ export const getStyles = (props: ILinkStyleProps): ILinkStyles => {
   return {
     root: [
       classNames.root,
-      theme.fonts.medium,
+      theme.fonts.inherit,
```

Comment:
Why is inherit on the theme, instead of just `fontWeight: 'inherit'`?

Reply:
I thought this would be a more elegant and themeable solution for components that needed to use the theme.fonts object, but didn't want to explicitly set font size or weight.

Reply:
What's the advantage to doing it this way vs `fontWeight: 'inherit'` and same for `fontSize`? Would a theme reasonably change the value of this?

Reply:
For instance, would there ever be a theme that would change `inherit` to be a different value? At that point it doesn't mean `inherit` anymore which seems like it'd be really confusing.

Reply:
You're probably right about that. I'll revert the theme changes and just add fontsize and weight: inherit.